org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServerWrapper.java

```diff
@@ -711,7 +711,15 @@ public LanguageServer getServer() {
 		}
 	}
 
-	/**
+  /**
+   * Returns the language server ID.
+   */
+  @NonNull
+  public String getServerId() {
+      return this.serverDefinition.id;
+  }
```

Comment:
Could we avoid adding API method here? you can directly call `wrapper.serverDefinition.id` from DocumentContentSynchronizer.

Reply:
thanks, done.